policy/watcher.go

```diff
@@ -486,31 +523,89 @@ func (w *watcher) defaultPolicyAction(action string) (map[string]Rule, error) {
 	return defPolicyActionMap, nil
 }
 
+func (w *watcher) defaultPolicyActionOut(action string) (map[string]Rule, error) {
+	defPolicyActionMap := make(map[string]Rule)
+
+	ruleName := "all.local.containers.out"
+	all := w.getAllLocalContainers()
+	isStateful := false
```

Comment:
If we want to allow only outgoing connections, this would become a stateful rule.